tests/libcxx/enc/CMakeLists.txt

```diff
@@ -41,14 +47,26 @@ function(add_libcxx_test_enc NAME CXXFILE)
         ${PROJECT_SOURCE_DIR}/3rdparty/libcxx/libcxx/fuzzing
         )
 
-    # Remove NDEBUG to enable the libcxx testsuite assertions in Release
-    target_compile_options(libcxxtest-${NAME}_enc PRIVATE -Wno-error --std=c++14 -UNDEBUG)
+    set_target_properties(libcxxtest-${NAME}_enc PROPERTIES
```

Comment:
-    target_compile_options(libcxxtest-${NAME}_enc PRIVATE -Wno-error --std=c++14 -UNDEBUG)

Please verify that this change did not make all the asserts in Release NOPs. Again, tests will PASS but test intent will be broken. Several of the libcxx tests use asserts to Pass/Fail the test and hence they need to be exercised.

Reply:
`-Wno-error` and `-UNDEBUG` are still defined the same way.

The only logic change here was relying on CMake to add `-std=c++14`, and it wouldn't compile otherwise. I'll test the release build in particular and post results.

Reply:
I've removed the change for `-std=c++14` and will do that in a separate PR to address all the uses of it.